code/modules/reagents/reagent_containers.dm

```diff
@@ -2,19 +2,29 @@
 	name = "Container"
```

Comment:
Unrelated changes

Reply:
Is more documentation on vars really going to kill us?